training/scip/summer-kickstart.rst

```diff
@@ -1,5 +1,5 @@
 ==================================
-Jun 2020 / FGCI Summer Kickstart
+June 2020 / FGCI Summer Kickstart
```

Comment:
```suggestion
June 2021 / FGCI Summer Kickstart
```
Can you verify these are the dates for 2021?

Reply:
thanks for the fixes. The dates for summer kickstat are updated for 2021 assuming the course will happen on the same week as last summer.